src/java.base/share/native/launcher/main.c

```diff
@@ -34,6 +34,14 @@
 #include "jli_util.h"
 #include "jni.h"
 
+#ifdef UNDEFINED_BEHAVIOR_SANITIZER
```

Comment:
I really do not like having to make source code changes to accommodate these kinds of tools.

Reply:
Yeah, it is unfortunate. However there is no other way to actually set the defaults nicely. The other alternative is to use environment variables, but they are easy to forget when invoking the launcher manually.

Reply:
Does the env-var override the src declaration, or the other way around?

Reply:
UBSAN_OPTIONS overrides the src declaration. I added to the comment mentioning that.